pym/bob/cmds/build.py

```diff
@@ -1095,22 +1097,20 @@ def doStatus(argv, bobRoot):
 
     def showStatus(package, recurse, verbose, done):
         checkoutStep = package.getCheckoutStep()
-        if not checkoutStep.getVariantId() in done:
+        if checkoutStep.isValid() and (not checkoutStep.getVariantId() in done):
             done.add(checkoutStep.getVariantId())
             print(">>", colorize("/".join(package.getStack()), "32;1"))
             oldCheckoutState = BobState().getDirectoryState(checkoutStep.getWorkspacePath(), {})
             if not os.path.isdir(checkoutStep.getWorkspacePath()):
```

Comment:
Note that `checkoutStep.getWorkspacePath()` may return `None` in release mode if the step was never run!